dev-starts3.sh

```diff
@@ -220,7 +220,8 @@ do
   pid_filename='/var/run/s3server.'$s3port'.pid'
   $valgrind_cmd s3server --s3pidfile $pid_filename \
            --motrlocal $local_ep:${motr_local_port} --motrha $ha_ep \
-           --s3port $s3port --fault_injection true $fake_params --loading_indicators --getoid true
+           --s3port $s3port --fault_injection true $fake_params --loading_indicators --getoid true \
+           --addb
```

Comment:
Pls confirm this is not a misprint.  If this is intended, please confirm we won't have problems with performance and disk usage in dev env with ADDB enabled.

Reply:
release configuration has addb enabled by default, I think it would be good to debug/develop on configuration as close to release as possible
addb has low performance impact and we usually dev-starts3 script is used not for performance tests but tests on dev VM
all addb data are removed on each jenkins-build script run or could be forced by jenkins-build --cleanup_only option, so I do not think there will be any significant extra disk usage.